### PR TITLE
Lenovo PHAB Plus fixes and updates

### PR DESCRIPTION
**Pull request by TipzRickyCheung**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,6 +90,7 @@ page on the EFIDroid wiki for an exact mapping of LK targets to SoCs.
 - Wileyfox Swift - crackling
 - Xiaomi Mi 4i - ferrari
 - Xiaomi Redmi 2 - wt86047, wt88047
+- Yamada EveryPad III
```
Comment (stephan-gh) [inline README.md:93]: Can you explain this in the commit message, if this was an intentional change?

File: dts/msm8916/msm8939-qrd-skuk.dts [hunk 2]
```diff
@@ -25,9 +25,9 @@
 		};
 	};
 
-	lenovo-pb1770m {
+	lenovo-phoenix {
 		model = "Lenovo PHAB Plus";
-		compatible = "lenovo,pb1770m", "qcom,msm8939", "lk2nd,device";
-		lk2nd,match-cmdline = "* mdss_mdp.panel=1:dsi:0:qcom,mdss_dsi_nt35532_1080p_skuk_video:1:none";
+		compatible = "lenovo,phoenix", "qcom,msm8939", "lk2nd,device";
+		lk2nd,match-cmdline = "* mdss_mdp.panel=1:dsi:0:qcom,mdss_dsi_nt35532_1080p_skuk_video:1:none androidboot.hw_id=110 ";
```
Comment (stephan-gh) [inline dts/msm8916/msm8939-qrd-skuk.dts:31]: It's probably easier and more reliable if you do

```suggestion
		lk2nd,match-cmdline = "* mdss_mdp.panel=1:dsi:0:qcom,mdss_dsi_nt35532_1080p_skuk_video:1:none*";
```

**Review by stephan-gh — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (TipzRickyCheung) [reply README.md:93]: This is the exact same device.

**Comment by TipzRickyCheung** [comment]
I've force pushed to my fork with the changes as you asked.

**Review by stephan-gh — APPROVED** [review]

**Comment by TipzRickyCheung** [comment]
I also got lk1st to boot, maybe hold for a sec?

**Comment by TipzRickyCheung** [comment]
lk1st doesn't light up properly.

**Comment by stephan-gh** [comment]
You need to be careful with lk1st, it's easy to brick your device with it. And the display often needs some special code to work properly.

**Comment by TipzRickyCheung** [comment]
> Member

Boots fine, able to get a vibration and can use fastboot to access it, oem lk_log shows it loads fine?
Nothing on the screen with a torch at the screen though.

**Comment by stephan-gh** [comment]
You'd likely need to add code to initialize the tps65132 regulator properly. There is no existing example code for this yet, the lk1st stuff is still very experimental (and **dangerous**, be prepared to recover your device using EDL in case of trouble).

**Comment by TipzRickyCheung** [comment]
> You'd likely need to add code to initialize the tps65132 regulator properly. There is no existing example code for this yet, the lk1st stuff is still very experimental (and **dangerous**, be prepared to recover your device using EDL in case of trouble).

I've experience a ton of 9006 encounters already and have prepared a full dump of the device in case it went wrong. It did boot into fastboot fine though and I can confirm that our device do use tps65132.

**Comment by stephan-gh** [comment]
Feel free to join the [chat on IRC/Matrix](https://github.com/msm8916-mainline/lk2nd#contact) if you would like to discuss the lk1st stuff further. I'm not comfortable discussing this through a GitHub comments at the moment because it's too much WIP.

**Comment by TipzRickyCheung** [comment]
> You'd likely need to add code to initialize the tps65132 regulator properly. There is no existing example code for this yet, the lk1st stuff is still very experimental (and **dangerous**, be prepared to recover your device using EDL in case of trouble).

I got it to work by adding `target_panel_reset_skuk(enable);` in target_display.c

**Comment by stephan-gh** [comment]
I applied the first commit, let's close this one for now. The lk1st stuff would be better suited for a separate PR once it is properly working.

> blank display probably because FAN53555 or NCP6335D.

As I wrote on Matrix: "the FAN53555/NCP6335D is the CPU regulator, it is not related to the display. There is probably something else wrong with the display :("

**Pull request closed without merging**